INDEX

```diff
@@ -482,6 +482,16 @@
 		"official": false,
 		"primary_pkg": "unifi5"
 	},
+	"urbackup": {
+		"MANIFEST": "urbackup.json",
+		"name": "UrBackup",
+		"icon": "urbackup.png",
```

Comment:
Can you please change this to `https://icons.freenas.org/community-icons/urbackup.png` as it's supposed to be a URI

Reply:
I've changed the url. How does it work? Who puts the image on that url?